### PR TITLE
feat: home page 

### PR DESCRIPTION
**Pull request by shivangisharma01**
image
image

### CODE REVIEW

File: Page/Wishlist/wishlist.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <header>
         <nav class="nav-container-box">
             <div class="nav-logo"><a href="/../index.html">Scentcy <i class="fa-solid fa-spray-can-sparkles"></i></a></div>
-            <input type="search" placeholder="   Search for items here" class="nav-input">
+            <input type="search" placeholder=" Search for items here" class="nav-input">
```
Comment (shashidwivedi) [inline Page/Wishlist/wishlist.html:21]: There's a leading space left here in placeholder. You can remove it.

**Review by shashidwivedi — APPROVED** [review]
Everything else looks good. Good to merge

File: Page/Wishlist/wishlist.html (hunk 1 above)
Comment (shivangisharma01) [reply Page/Wishlist/wishlist.html:21]: okay

**Comment by hussiantabha** [comment]
Looks good to merge

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            </div>
+
+        </div>
+    </div>
+
+
+        </main>
+    
+    </body>
+
```
Comment (nishasen) [inline index.html:96]: Remove the extra spaces

**Review by nishasen — APPROVED** [review]
Added a single comment, else everything looks good

File: index.html (hunk 2 above)
Comment (shivangisharma01) [reply index.html:96]: ok sure.

**Pull request merged**